docs/contacts/activity.md

```diff
@@ -0,0 +1,9 @@
+# Activities
+
+Activities are interactions you’ve had with an individual and are recorded on that record’s timeline. Activities include emails, calls, tasks, notes and meetings. Activities also have the ability to be filtered so that team members know last communication, ticket replies, or a variety of other helpful updates.
```

Comment:
`Activities are interactions you’ve had with an individual`

Edit: Activities are interactions ~~you’ve~~ **that a team member has** had with a~~n individual~~ **contact** 

`so that team members know last communication`
Edit: so that team members ~~know~~ **can view**